invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -291,6 +291,9 @@ def load_custom_fields():
             if field_error_label:
                 error_labels[f"custom_fields.{field['field']}"] = field_error_label
             if getattr(field_instance, "relation_cls", None):
+                sort_by = field.get("props", {}).get("sort_by")
+                if sort_by:
+                    field_instance.sort_by = sort_by
```

Comment:
To be picked up by this: https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/services/custom_fields/vocabulary.py#L90